docs/tutorials/dask_client.ipynb

```diff
@@ -0,0 +1,247 @@
+{
```

Comment:
<p>Whenever you're using the sphinx directive for in-site links, you have to do the annoying raw cells. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I'd actually already added an addendum to the issue description! Great minds :^)